.github/workflows/validate-changelog.yaml

```diff
@@ -11,12 +11,17 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Lint changelog file
-        uses: avto-dev/markdown-lint@v1
-        with:
-          rules: /lint/rules/changelog.js
-          config: /lint/config/changelog.yml
-          args: ./CHANGELOG.md
+      - name: Setup Helm
+        uses: azure/setup-helm@v3.5
+
+      - name: Setup ct (chart testing)
+        uses: helm/chart-testing-action@v2.4.0
+
+      - name: Run Helm chart-testing (lint)
+        run: make lint
```

Comment:
this looks unrelated, please fork to another PR

Reply:
done